jobserver/migrations/0008_coredeveloper_to_staffareaadministrator.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 5.1.1 on 2024-10-02 11:00
+
+from django.db import migrations
+
+from jobserver.authorization.roles import CoreDeveloper, StaffAreaAdministrator
+
+
+def change_user_role(apps, from_role, to_role):
+    User = apps.get_model("jobserver", "User")
+    users = User.objects.filter(roles__contains=from_role)
+    for user in users:
+        user.roles.remove(CoreDeveloper)
+        user.roles.append(StaffAreaAdministrator)
```

Comment:
I think you mean...

```suggestion
        user.roles.remove(from_role)
        user.roles.append(to_role)
```

because otherwise the reverse migration will fail with...

```
ValueError: list.remove(x): x not in list
```

Reply:
I absolutely do mean that! Thanks for catching that